ecms_base/themes/custom/ecms/ecms.theme

```diff
@@ -87,7 +87,12 @@ function ecms_theme(array $existing, string $type, string $theme, string $path):
 function ecms_preprocess_paragraph__media_item(array &$variables): void {
   $paragraph = $variables['paragraph'];
 
-  $media_type = $paragraph->field_media_item->entity->bundle();
+  if (isset($paragraph->field_media_item->entity)) {
+    $media_type = $paragraph->field_media_item->entity->bundle();
+  }
+  else {
+    return;
+  }
```

Comment:
Instead of the if/else block, it will be cleaner to setup a guard clause instead:
```
if ($paragraph->field_media_item->isEmpty()) {
  return;
}
```